app/scripts/pages/connections.js

```diff
@@ -1,8 +1,7 @@
-'use strict'
 var React = require('react')
-var ConnectionList = require('../views/connectionlist.jsx')
-var Globe = require('../views/globe.jsx')
-var getLocation = require('ipfs-geoip').lookupPretty
+var ConnectionList = require('../views/connectionlist')
```

Comment:
I'm curious, no `const` or arrow functions this time? Any specific reason why or are you saving that for another PR?